build/tools.mak

```diff
@@ -27,7 +27,4 @@ define make_with_tracking_file
 	$(_hide_cmds)$(call log_command,touch $1)
 endef
 
-# Delete the file whose absence will trigger tox to run
```

Comment:
why did you get rid of this?

Reply:
We don't use it any more. This was from when make ran tox to test. We no longer do that. Instead, we use tox to run make and then we test.

This should have been removed when we made those other changes.